server/admin/server_test.go

```diff
@@ -25,13 +29,13 @@ func init() {
 }
 
 var (
-	_ SvrCore = (*TCore)(nil)
+	// Check that *dexsrv.DEX satisfies SvrCore.
+	_ SvrCore = (*dexsrv.DEX)(nil)
```

Comment:
Maybe remove.  It's nice to ensure this, but the dcrdex compilation will fail anyway.  And the import is otherwise unused.